packages/rum/src/getAPMDocumentData.ts

```diff
@@ -0,0 +1,89 @@
+import { findCommaSeparatedValue } from '@datadog/browser-core'
+
+interface APMDocumentData {
```

Comment:
```suggestion
interface DocumentTraceData {
```
I would be in favor of replacing all APM references by trace for consistency